cogs/utils/db.py

```diff
@@ -172,6 +172,21 @@ async def set_member(guildID, memberID, option, value):
     await conn.close()
 
 
+# Returns members who have birthday on a given date
+async def get_members_bday_equals_date(date):
+    conn = await db_connect()
+    c = await conn.cursor()
+    sql = f"""
+    SELECT MIN(member_id) AS member_id, *
+    FROM members
+    WHERE birthday LIKE '%{date.strftime('%m')}-{date.strftime('%d')}%'
+        OR birthday LIKE '%{date.month}-{date.day}%';"""
```

Comment:
Inserting variables through fstring into SQLite could potentially be a point of SQL injection vulnerability, according to the python docs: https://docs.python.org/3/library/sqlite3.html

This can be avoided by using `?` in place of the values in the SQL and passing the variables through the `.execute()` statement.